src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpTraceHelper.cs

```diff
@@ -33,7 +33,17 @@ public static bool IsTraceEnabled()
             return s_TraceEnabled;
         }
 
-        public static void TraceCallbackStatus(string message, IntPtr handle, bool invokeCallback, uint status)
+        public static void Trace(string format, params object[] args)
```

Comment:
I didn't realize that using `params object[]` would cause that extra allocation. I think for now I'll just revise the method to have explicit arguments. That is what methods like `string.Format` do for common overload signatures in addition to the `params object[]` argument.